curvefs/src/mds/fs_manager.h

```diff
@@ -151,8 +151,9 @@ class FsManager {
                            FsInfo* fsInfo);
 
  private:
-    bool IsPreviousCreateUnComplete(const std::string& fsName, FSType fsType,
-                                    uint64_t blocksize, const FsDetail& detail);
+    // return 1: exist; 2: uncomplete; 0: neither
```

Comment:
maybe `0: ExactlySame, 1: uncomplete, -1: neither` is better?

Reply:
> maybe `0: ExactlySame, 1: uncomplete, -1: neither` is better?

fixed